portal/server/api-ssrf.test.js

```diff
@@ -0,0 +1,410 @@
+const MockAdapter = require('axios-mock-adapter');
+const axios = require('axios');
+require('dotenv').config();
+
+const api = require('./api');
+
+const { PORTAL_API_URL } = process.env;
+
+describe('api', () => {
+  const mock = new MockAdapter(axios);
+
+  const invalidId = '../../../etc/passwd';
```

Comment:
Can we also add following for invalid ID assertions:

* `127.0.0.1`
* []
* {}